environments/values-usdfdev.yaml

```diff
@@ -0,0 +1,71 @@
+environment: usdfdev
+fqdn: usdf-rsp-dev.slac.stanford.edu
+vault_path_prefix: secret/rubin/usdf-rsp-dev
+# butlerRepositoryIndex: "s3://butler-us-central1-repo-locations/data-repos.yaml"
+repoURL: https://github.com/slaclab/phalanx.git
+
+alert_stream_broker:
```

Comment:
All the underscores in application names should now be dashes after a recent change. So `alert-stream-broker`, `cert-manager`, etc.

Reply:
fixed in latest commit

Reply:
actually, I forgot the prod version..will correct that as well.